magento/magento2
# Adds some more colors to the output of certain setup cli commands

### Description (*)
While trying to figure out the sort order in which certain database migration scripts ran while executing `bin/magento setup:upgrade`, I found the output a bit hard to read.
So I've added a different color to the output of certain steps.

Before:
<img width="404" alt="before" src="https://user-images.githubusercontent.com/85479/88589484-86c5d400-d059-11ea-90d1-e98529470066.png">

After:
<img width="404" alt="after" src="https://user-images.githubusercontent.com/85479/88589512-8e857880-d059-11ea-993a-81fdcf5662dc.png">

This also influences some other `bin/magento setup:xxx` commands but it doesn't seem to degrade them in my opinion (but I haven't tested all of them).

### Fixed Issues (if relevant)
None that I could find

### Manual testing scenarios (*)
1. Execute the `bin/magento setup:upgrade` command and look if the colors used in the output make sense
2. Execute all other console commands that use the ConsoleLoggerInterface class and see if the colors used in the output make sense

### Questions or comments
If you want different colors or if there are better idea's, let me know!
I've just re-used the "metadata" color which was already being used.

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29335: Adds some more colors to the output of certain setup cli commands
